docs/docs/deploy/environment-variables.mdx

```diff
@@ -0,0 +1,131 @@
+---
+id: environment-variables
+title: Adding Environment Variables
+abstract: |
+    Rasa offers advanced configuration through environment variables.
+    Find the full reference here.
+---
+
+import RasaProLabel from "@theme/RasaProLabel";
+
+To add extra environment variables to your containers, add the
+environment variable to an `extraEnvs` array on the service in your `values.yml`. These
+values can be provided directly or refer to [secrets](../production/secrets-managers.mdx).
+The `extraEnvs` parameter is available for all services.
+For example:
+
+```yaml
+# rasa: Settings common for all Rasa containers
+rasa:
+  # other configured parameters
+  # ...
+  extraEnvs:
+  - name: "LOG_LEVEL"
+    value: "warning"
+```
+
+## Rasa Pro and Rasa Open Source
+
+When running Rasa Pro, the `RASA_PRO_LICENSE` environment variable needs to contain
+the Rasa Pro license key.
+
+### Backing services
+
+| Environment Variable              | Description                                                                                                                                                                                                           | Default           |
+| :-------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------- |
+| `POSTGRESQL_SCHEMA`               | The postgres schema for the [`SQLTrackerStore`](../production/tracker-stores.mdx#sqltrackerstore).                                                                                                                    | `"public"`        |
+| `POSTGRESQL_POOL_SIZE`            | The limit of the number of open connections for the [`SQLTrackerStore`](../production/tracker-stores.mdx#sqltrackerstore).                                                                                            | `50`              |
+| `POSTGRESQL_MAX_OVERFLOW`         | The [maximum overflow](https://docs.sqlalchemy.org/en/13/core/pooling.html#sqlalchemy.pool.QueuePool.params.max_overflow) of pool size for the [`SQLTrackerStore`](../production/tracker-stores.mdx#sqltrackerstore). | `100`             |
```

Comment:
Not actionable in this PR, but these should be available configurations in the endpoints.yaml. If you think the same, I can create a ticket for this

Reply:
yeah maybe. This should be a feature request though :)